install/kubernetes/cilium/templates/NOTES.txt

```diff
@@ -5,8 +5,12 @@
     If you have an issues please refer to the CNP Validation section in the upgrade guide.
 {{- else if (not (.Values.config.enabled))}}
     You have preserved the configMap and successfully installed {{ title .Chart.Name}}.
-{{- else if (and (.Values.global.hubble.enabled) (.Values.global.hubble.ui.enabled)) }}
-    You have successfully installed Cilium with hubble-ui.
+{{- else if (and (.Values.global.hubble.enabled) (.Values.global.hubble.relay.enabled)) }}
+    {{- if (.Values.global.hubble.ui.enabled) }}
+        You have successfully installed Cilium with Hubble Relay and Hubble UI.
```

Comment:
nits: should _Cilium_ be {{ title .Chart.Name }} for consistency of messages with other cases as well ?